connector/src/main/java/org/jboss/as/connector/deployers/ds/processors/DsXmlDeploymentParsingProcessor.java

```diff
@@ -90,6 +89,7 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
                 DataSources dataSources = parser.parse(xmlStream);
 
                 if (dataSources != null) {
+                    ConnectorLogger.ROOT_LOGGER.deprecated();
```

Comment:
Since this is in a loop, it should be guarded by a boolean var so it only logs once if there's more than one -ds.xml.